pulp_smash/tests/platform/api_v2/simple_repo_test.py

```diff
@@ -0,0 +1,174 @@
+# coding=utf-8
+"""Test for basic repo creating functionality.
+"""
+from __future__ import unicode_literals
+
+from pulp_smash.resources.platform.api_v2.api import Repository, Task
+from pulp_smash.resources.platform.api_v2.api import ERROR_KEYS
+from unittest2 import TestCase
+
+
+class RepoCreateSuccessTestCase(TestCase):
+    """Tests for successfull repo creating functionality.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Create repo on pulp server
+        """
+        cls.repo = Repository(id=cls.__name__)
```

Comment:
What happens if I cancel this test before `tearDownClass` runs, or if the repository deletion fails? In that case, I am unable to run this test again, because a repository with an identical ID already exists. Using a random string for the repository ID lets me avoid this issue.